src/app/js/fields/FieldInternalState.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import translate from 'redux-polyglot/translate';
+import { Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import HomeIcon from '@material-ui/icons/Home';
+import DescriptionIcon from '@material-ui/icons/Description';
+import BarChartIcon from '@material-ui/icons/BarChart';
+import theme from '../theme';
+import classnames from 'classnames';
+
+const useStyles = makeStyles({
+    container: {
+        display: 'flex',
+        flexDirection: 'row',
+        alignItems: 'center',
+        marginRight: '1rem',
+    },
+    state: {
+        padding: '0.5rem',
+        border: '1px solid black',
```

Comment:
I've noticed that the black is not the right color, I think the color should be theme.black.secondary